pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -359,7 +359,8 @@ First In, First Out: Files will be uploaded in the order they were queued.</prop
               <object class="GtkRadioButton" id="_use_upload_bandwidth_radio">
                 <property name="visible">True</property>
                 <property name="hexpand">True</property>
-                <property name="label" translatable="yes">Queue uploads if total transfer speed reaches (KiB/s):</property>
+                <property name="label" translatable="yes">Allocate upload slots until total transfer speed reaches (KiB/s):</property>
```

Comment:
I think the length of the string was the original issue I had with it, but `total transfer speed -> total speed` seems like a good compromise. Maybe we could also move the Queue Behavior section after Upload Speed Limits to make the relationship more obvious.